src/Toggle/Entity/ActivatableToggle.php

```diff
@@ -0,0 +1,8 @@
+<?php
+namespace Clearbooks\Labs\Toggle\Entity;
+
+interface ActivatableToggle
```

Comment:
I guess this is where all the tendrils of the system will need to hook into making toggles active.